JS/Lesson34/goksel/recipe-app/src/modules/NewRecipe/NewRecipeInformations/index.jsx

```diff
@@ -0,0 +1,69 @@
+export const MealCategory = [
+    {
+        title: "Dessert",
+        id: "dessert"
+    },
+    {
+        title: "Main Dish",
+        id: "mainDish"
+    },
+    {
+        title: "Side Dish",
+        id: "sideDish"
+    }
+]
+
+
+export const MealArea = [
+    {
+        title : "British",
+        id : "british"
+    },
+    {
+        title : "Canadian",
+        id : "canadian"
+    },
+    {
+        title : "Russia",
+        id : "russia"
+    },
+    {
+        title : "India",
+        id : "india"
+    },
+    {
+        title : "Greece",
+        id : "greece"
+    },
+    {
+        title : "Turkey",
+        id : "turkey"
+    },
+]
+
+
+
+function handlestrIngredientsAndMeasures(data) {
+
+    const ingredientsAndMeasures = data.split(",");
+    const ingredientsAndMeasuresObj = ingredientsAndMeasures.map((ingredientAndMeasure) => {
+
+        const [ingredient, measure] = ingredientAndMeasure.trim().replace(/\s+/g, " ").split("-");
+        return { ingredient, measure };
+    })
+    return ingredientsAndMeasuresObj;
```

Comment:
Good try so far but you need to put a bit more effort here to make the data compatible with the API data as the goal is to merge recipes created by a user with recipes that we fetch from API. 
You need to convert these data into this format: 
{
			"strIngredient1": "Plain Flour",
			"strIngredient2": "Caster Sugar",
			"strIngredient3": "Butter",
			"strIngredient4": "Braeburn Apples",
			"strIngredient5": "Butter",
			"strIngredient6": "Demerara Sugar",
			"strIngredient7": "Blackberrys",
			"strIngredient8": "Cinnamon",
			"strIngredient9": "Ice Cream",
			"strIngredient10": "",
			"strIngredient11": "",
			"strIngredient12": "",
			"strIngredient13": "",
			"strIngredient14": "",
			"strIngredient15": "",
			"strIngredient16": "",
			"strIngredient17": "",
			"strIngredient18": "",
			"strIngredient19": "",
			"strIngredient20": "",
			"strMeasure1": "120g",
			"strMeasure2": "60g",
			"strMeasure3": "60g",
			"strMeasure4": "300g",
			"strMeasure5": "30g",
			"strMeasure6": "30g",
			"strMeasure7": "120g",
			"strMeasure8": "\u00bc teaspoon",
			"strMeasure9": "to serve",
			"strMeasure10": "",
			"strMeasure11": "",
			"strMeasure12": "",
			"strMeasure13": "",
			"strMeasure14": "",
			"strMeasure15": "",
			"strMeasure16": "",
			"strMeasure17": "",
			"strMeasure18": "",
			"strMeasure19": "",
			"strMeasure20": "",
}